tests/test_time.py

```diff
@@ -29,7 +29,14 @@ def __init__(self, unique_id, model):
         self.steps = 0
         self.advances = 0
 
+    def kill_agent(self):
```

Comment:
`kill_other_agent` sounds clearer. Also, I think removing just 1 agent is sufficient.